src/System.Private.Xml/tests/XmlSchema/XmlSchemaSet/TC_SchemaSet_AnyAttribute.cs

```diff
@@ -188,6 +210,43 @@ public void v2(string ns1, string ns2, string attrNs, int expectedError)
             xss.Compile();
 
             Assert.Equal(expectedError, _errorCount);
+
+            // Full framework does not set the namespace property for intersections and unions
+            if (!PlatformDetection.IsFullFramework && expectedNs != null)
+            {
+                XmlSchemaAnyAttribute attributeWildcard = ((XmlSchemaComplexType)xss.GlobalTypes[new XmlQualifiedName("t1", attrNs)]).AttributeWildcard;
+                CompareWildcardNamespaces(expectedNs, attributeWildcard.Namespace);
+            }
+        }
+
+        [Theory]
+        //[Variation(Desc = "ns - ##any, allow any attribute")]
+        [InlineData("##any", "ns1", "##any")]
+        //[Variation(Desc = "ns - ##other, not allow ns1 attribute")]
+        [InlineData("##other", "ns1", "##other")]
+        //[Variation(Desc = "ns - ns1 ns2, allow ns1 and ns2 attribute")]
+        [InlineData("ns1 ns2", "ns1", "ns1 ns2")]
+        //[Variation(Desc = "##targetNamespace, allow current ns")]
+        [InlineData("##targetNamespace", "ns1", "##targetNamespace")]
+        public void v3(string ns, string attrNs, string expectedNs)
+        {
+            XmlSchemaSet xss = new XmlSchemaSet();
+            xss.XmlResolver = new XmlUrlResolver();
+            xss.ValidationEventHandler += new ValidationEventHandler(ValidationCallback);
+            xss.Add(GetSimpleSchema(ns, attrNs));
+            xss.Compile();
+
+            XmlSchemaAnyAttribute attributeWildcard = ((XmlSchemaComplexType)xss.GlobalTypes[new XmlQualifiedName("t", attrNs)]).AttributeWildcard;
+            CompareWildcardNamespaces(expectedNs, attributeWildcard.Namespace);
+        }
+
+        private static void CompareWildcardNamespaces(string expected, string actual)
```

Comment:
would it make sense to instead split, order and join back to string instead? Alternatively Split, order and just compare IEnumerable with Assert.Equal which I believe should correctly compare elementwise

Reply:
Fair point. I've rewritten it to string compares which does improve readability.

Reply:
The failing tests are now in `System.Net.Sockets.Tests` and `System.Reflection.Emit.ILGeneration.Tests`. I don't believe this can be caused by the last commit on this PR as that only changed a test file.